spec/models/candidate_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Candidate, type: :model do
+
+  it "has a valid factory" do
+    expect(build(:candidate)).to be_valid
+  end
+
+  # Lazily loaded to ensure it's only used when it's needed
+  let(:cand) { build(:candidate) }
+  let(:saved) { create(:candidate) }
+
+  describe "ActiveModel validations" do
+    # Basic validations
+    it { expect(cand).to validate_uniqueness_of(:profile_id).scoped_to(:post_id,:election_id)
+                             .with_message('har redan en likadan kandidatur')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 25) spaces for indenting an expression spanning multiple lines.